Client/mods/deathmatch/logic/luadefs/CLuaEngineDefs.cpp

```diff
@@ -705,6 +707,35 @@ int CLuaEngineDefs::EngineSetModelLODDistance(lua_State* luaVM)
     return 1;
 }
 
+int CLuaEngineDefs::EngineResetModelLODDistance(lua_State* luaVM)
+{
+    SString          strModel = "";
+    CScriptArgReader argStream(luaVM);
+    argStream.ReadString(strModel);
+
+    if (!argStream.HasErrors())
```

Comment:
As per #821 we should do a hard error here like so:
```cpp
if (argStream.HasErrors())
    return luaL_error(luaVM, argStream.GetFullErrorMessage());
```
and then move the passing code after that.